Resolutions/SkylandersGiants_Resolution/rules.txt

```diff
@@ -0,0 +1,111 @@
+[Definition]
+titleIds = 000500001010D700,0005000010116000
```

Comment:
Small nit: add space between the comma and the next title id

Reply:
Done.